sdk/batch/Azure.ResourceManager.Batch/tests/TestCase/BatchAccountCertificateOperationTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Batch.Models;
+using Azure.ResourceManager.Batch.Tests.Helpers;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Batch.Tests.TestCase
+{
+    public class BatchAccountCertificateOperationTests : BatchManagementTestBase
+    {
+        public BatchAccountCertificateOperationTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        private async Task<BatchAccountCertificateResource> CreateAccountResourceAsync(string accountName)
+        {
+            var container = (await CreateResourceGroupAsync()).GetBatchAccounts();
+            var input = ResourceDataHelper.GetBatchAccountData();
+            var lro = await container.CreateOrUpdateAsync(WaitUntil.Completed, Recording.GenerateAssetName("testaccount"), input);
```

Comment:
please rename the variable container here to collection

Reply:
fixed